ui/src/containers/SampleListViewContainer.jsx

```diff
@@ -526,43 +538,67 @@ export default function SampleListViewContainer() {
   }
 
   function getSynchronizationDropDownList() {
-    if (loginData.limsName.length === 1) {
-      return (
+    return (
+      <Dropdown as={ButtonGroup}>
         <TooltipTrigger
           id="sync-samples-tooltip"
-          tooltipContent={`Synchronise sample list with ${loginData.limsName[0].name}`}
+          tooltipContent={`Synchronise sample list with ${loginData.limsName[0].name},
+          and apply filter to only show with LIMS samples`}
         >
           <Button
-            className="nowrap-style"
             variant="outline-secondary"
+            className="nowrap-style"
             onClick={() => handleSyncSamples(loginData.limsName[0].name)}
           >
             <i className="fas fa-sync-alt" style={{ marginRight: '0.5em' }} />
             Get Samples
           </Button>
         </TooltipTrigger>
-      );
-    }
-    return (
-      <Dropdown>
-        <Dropdown.Toggle variant="outline-secondary" id="dropdown-lims">
-          <i className="fas fa-sync-alt" style={{ marginRight: '0.5em' }} />{' '}
-          Synchronize with
-        </Dropdown.Toggle>
+        {(loginData.limsName.length > 1 || showGetSamplesFromSC) && (
+          <Dropdown.Toggle
+            split
+            variant="outline-secondary"
+            id="dropdown-split-samples"
+            title="Other LIMS Options"
+          />
+        )}
+
         <Dropdown.Menu>
-          {loginData.limsName.map((lims) => (
-            <TooltipTrigger
-              key={lims.name}
-              tooltipContent={`Synchronise sample list with ${lims.name}`}
-            >
+          {showGetSamplesFromSC && (
+            <TooltipTrigger tooltipContent="get samples from sample changer">
               <Dropdown.Item
-                key={lims.name}
-                onClick={() => handleSyncSamples(lims.name)}
+                onClick={() => getSamplesFromSC()}
+                variant="outline-secondary"
+                className="nowrap-style"
               >
-                {lims.name}
+                Get Samples from SC
               </Dropdown.Item>
             </TooltipTrigger>
-          ))}
+          )}
+          {loginData.limsName.length > 1 && <Dropdown.Divider />}
+          {loginData.limsName.map((lims, lindex) => {
```

Comment:
```suggestion
          {loginData.limsName.slice(1).map((lims, lindex) => {
```

Otherwise you can split the LIMS beforehand:

```js
const firstLims = ...
const extraLims = ...
```